examples/spin-timer/spin-timer.wit

```diff
@@ -1 +1 @@
-handle-timer-request: func(msg: string) -> string
+handle-timer-request: func()
```

Comment:
nit: should `handle-timer-request` take a `u64` or `string` encoding of the time that triggered the invocation? Understandably for an example this isn't too important so feel free to ignore.

Reply:
Probably it should - I was sort of trying to avoid wading into the TIME WHAT EVEN IS IT complexities of what the "current time" is and how to represent it across the WIT boundary.  If you don't mind, I'd prefer to punt on that for this PR, though I agree it would be good to show how to pass inputs and process results from the guest.